### PR TITLE
bottles: 2021.7.14-treviso -> 2021.9.14-treviso

### PR DESCRIPTION
**Pull request by asbachb**
Added new dependency `patool` as it's now needed during runtime.

###### Motivation for this change
Update to latest release.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
```
$ git -c fetch.prune=false fetch --force https://github.com/NixOS/nixpkgs master:refs/nixpkgs-review/0
remote: Enumerating objects: 30, done.
remote: Counting objects: 100% (24/24), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 30 (delta 15), reused 14 (delta 13), pack-reused 6
Unpacking objects: 100% (30/30), 224.55 KiB | 782.00 KiB/s, done.
From https://github.com/NixOS/nixpkgs
   9f761047cc4..b7255cd8b96  master     -> refs/nixpkgs-review/0
$ git worktree add /home/asbachb/.cache/nixpkgs-review/rev-df00de53009d941f5417aa298761f558da758a5b-dirty/nixpkgs b7255cd8b96994ebfc20810f3cad4b6e871af7f4
Preparing worktree (detached HEAD b7255cd8b96)
Updating files: 100% (27893/27893), done.
HEAD is now at b7255cd8b96 Merge pull request #138542 from figsoda/cargo-tally
$ nix-env --option system x86_64-linux -f /home/asbachb/.cache/nixpkgs-review/rev-df00de53009d941f5417aa298761f558da758a5b-dirty/nixpkgs -qaP --xml --out-path --show-trace
No diff detected, stopping review...
$ git worktree prune
```
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by happysalada** [comment]
Happy to help you get this merged.
The diff looks good.
There seems to be a conflict, could you have a look at it ?

File: pkgs/applications/misc/bottles/default.nix [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
   pname = "bottles";
-  version = "2021.7.14-treviso";
+  version = "2021.9.14-treviso";
 
   src = fetchFromGitHub {
     owner = "bottlesdevs";
     repo = pname;
     rev = version;
-    sha256 = "0xhfk1ll8vacgrr0kkhynq4bryjhfjs29j824bark5mj9b6lkbix";
+    hash = "sha256:0wdb6pc9gl6fnmd500smsq303snncaim284wgz7isjwhmwmfyw8m";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/bottles/default.nix:17]:
```suggestion
    sha256 = "0wdb6pc9gl6fnmd500smsq303snncaim284wgz7isjwhmwmfyw8m";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Master got an update in the meanwhile. Please rebase.

**Comment by asbachb** [comment]
As the there was also an upstream update I'll open a new PR.

**Pull request closed without merging**